### PR TITLE
spring-boot. points 1,2

### PR DESCRIPTION
**Pull request by MaVaYuran**
Hello. Sorry, was busy

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -8,102 +8,51 @@
...
+        <relativePath/>
+    </parent>
 
     <properties>
         <maven.compiler.source>21</maven.compiler.source>
         <maven.compiler.target>21</maven.compiler.target>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 
-        <spring.version>6.1.12</spring.version>
+<!--        <spring.version>6.1.12</spring.version>-->
```
Comment (Alexmansar) [inline pom.xml]: remove it please

**Review by Alexmansar — CHANGES_REQUESTED** [review]
@MaVaYuran please, fix small comment
and what about this part?
image

**Comment by MaVaYuran** [comment]
Alexandr, pls could you say something?

File: src/main/java/by/mariayuran/springboot/config/SecurityConfig.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+@Configuration
+@EnableWebSecurity
+public class SecurityConfig {
+
+    @Bean
+    public SecurityFilterChain securityFilterChain(HttpSecurity http) throws Exception {
+        http
+                .authorizeHttpRequests(authorize -> authorize
+                        .requestMatchers("/tickets**").authenticated()
```
Comment (Alexmansar) [inline src/main/java/by/mariayuran/springboot/config/SecurityConfig.java]: fis it as we chat in telegram

**Review by Alexmansar — CHANGES_REQUESTED** [review]
@MaVaYuran Plese, fix it and merge

until you click on the button again, no review request will come to me. 
image

**Review by Alexmansar — APPROVED** [review]
@MaVaYuran Hi) please, merge it

**Pull request merged**